code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -3831,15 +3823,6 @@
 	description = "Popular with the sailors. Not very popular with everyone else."
 	color = "#664300" //rgb: 102, 67, 0
 
-/datum/reagent/ethanol/deadrum/on_mob_life(var/mob/living/M)
-
-	if(..()) return 1
-
-	M.dizziness += 5
-
-	if(volume > REAGENTS_OVERDOSE)
```

Comment:
Remove just this line and the one below it for deadrum and absinthe and you have yourself a good PR. 

Regular ethanol only damages you above 450u of consumption, but both absinthe and deadrum start damaging you above reagents overdose, which occurs at just 30u. So remove just those and this could actually get merged.